src/app/shared/components/profile-image/profile-image.component.html

```diff
@@ -0,0 +1,29 @@
+<input
+  *ngIf="viewOrEdit === 'edit'"
```

Comment:
for view or edit it would be better to have a boolean flag that allows editing, maybe call it "isEditable"?

Reply:
isEditable should default to false, and as such, should not be defined and passed explicitly by each component that uses this